### DISCUSSION TITLE
Forth on Neo6502

### DISCUSSION THREAD

**Discussion by zbyti** [paulscottrobson/neo6502-firmware#187] [Ideas]
https://github.com/SamCoVT/TaliForth2

Perhaps it will be straightforward to port this. If nobody else takes on the task, I might give it a try ;)

**Comment by paulscottrobson** [comment]
Should be fairly easy, it's designed to be ported I think.

**Comment by zbyti** [comment]
https://github.com/SamCoVT/TaliForth2/blob/master-64tass/platform/platform-neo6502.asm

looks like it's done already :D

**Comment by astralaster** [comment]
Forth is now ported and works.
